comp/remote-config/bundle.go

```diff
@@ -17,5 +18,6 @@ import (
 func Bundle() fxutil.BundleOptions {
 	return fxutil.Bundle(
 		rcclientimpl.Module(),
+		rcstatusimpl.Module(),
 	)
 }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** I see the system probe is supplying `rcclientimpl.Module()` still, so it's going to be missing the `rcstatusimpl.Module()`. Does it need the bundle to get the status component as well? I'm not sure if it's needed or not.‏

Reply:
The system probe agent does not have the status command. We could add it, but it would be of no use 😄